bots/notify/src/test/java/org/openjdk/skara/bots/notify/mailinglist/MailingListNotifierTests.java

```diff
@@ -79,17 +81,19 @@ void testMailingList(TestInfo testInfo) throws IOException {
                                              .build();
             updater.attachTo(notifyBot);
 
-            // No mail should be sent on the first run as there is no history
+            // One mail should be sent on first commit
```

Comment:
So all these tests needed to be modified because the test repos are so small the fall below the threshold?

Reply:
Yes, our existing tests all just have few commits, and the commit counts are all  below the threshold, so I added a test with a repo which has 'long' history.